app/src/main/java/com/pwhs/quickmem/presentation/app/search_result/SearchResultUiAction.kt

```
@@ -1,3 +1,8 @@
 package com.pwhs.quickmem.presentation.app.search_result
 
-sealed class SearchResultUiAction()
+sealed class SearchResultUiAction() {
```

detekt

[warning] An empty default constructor can be removed.

(detekt.empty-blocks.EmptyDefaultConstructor)